recipes/utils/e2fsprogs/0002-e2fsprogs-disable-resource-tracking.patch

```diff
@@ -0,0 +1,28 @@
+diff -ur a/e2fsck/e2fsck.h b/e2fsck/e2fsck.h
+--- a/e2fsck/e2fsck.h	2024-12-18 17:06:42.712503044 +0100
++++ b/e2fsck/e2fsck.h	2024-12-18 17:27:10.189747773 +0100
+@@ -145,7 +145,9 @@
+ 
+ struct encrypted_file_info;
+ 
+-#define RESOURCE_TRACK
++// Disabled as this functionality needs sbrk() which is not available
++// universally.
++//#define RESOURCE_TRACK
```

Comment:
Could we leave this enabled by default and only disable it if `__l4re__` is defined?

Reply:
Ack, done.